### DISCUSSION TITLE
RA/DEC (J.2000) vs On Date values displayed by Stellarium relative to Solar System planets.

### DISCUSSION THREAD

**Discussion by aeromoe** [Stellarium/stellarium#1892] [Q&A]
Hello - the RA/DEC values listed when each planet is selected differ by many degrees.  For December 1, 2021 Mercury shows 252 degrees for J.2000 and 236 degrees "On date" (minutes/seconds omitted).    Each of the other planets in the Solar System has a similar disparity.  Why is this?

For context, I'm creating a large wall chart where I use small-headed push pins to plot the monthly positions of the planets as seen from above as they orbit the sun.  In searching for the easiest way to determine these positions, I found the positions I need are the Heliocentric planetary longitudes and they correspond to the values in the subject of this discussion.  

At the scale of my chart, the orbit of Neptune is about 6.7 feet (just over 2 meters) in diameter, and the outer planet orbits are simple circles and scale distances from each other.  The inner planets can be fairly accurately positioned by taking the positions from the solar system chart on the Heavens-Above website, modified to fit the scale of the all the orbits.

Thanks for your time.

**Comment by alex-w** [comment]
Please read https://en.wikipedia.org/wiki/Epoch_(astronomy)

**Comment by gzotti** [comment]
For this wall chart you indeed need the heliocentric longitudes, which are however not available. When you are looking at the solar system from above e.g. from the Solar System Observer, the displayed coordinates are likely meaningless in practice because they refer to the observer location. 

You could place yourself in the center of the sun, i.e. move to the sun as observing location, switch off ground and topocentric correction, and read ecliptical longitudes in the J2000 frame.

**Comment by aeromoe** [comment]
Hi George....yes, that is exactly what I did...Sun is the observing location and the J.2000 values closely match reality.  I was just curious why the "On Date" values vary by so much.   
The J.2000 values are good and close enough for my display scale.

I received a link to the Wiki article concerning epochs which very technically explains it.  
Thank you
Moe

Sent from the all new AOL app for Android 
 
  On Sat, Sep 18, 2021 at 12:31 AM, Georg ***@***.***> wrote:   


For this wall chart you indeed need the heliocentric longitudes, which are however not available. When you are looking at the solar system from above e.g. from the Solar System Observer, the displayed coordinates are likely meaningless in practice because they refer to the observer location.

You could place yourself in the center of the sun, i.e. move to the sun as observing location, switch off ground and topocentric correction, and read ecliptical longitudes in the J2000 frame.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by gzotti** [comment]
Ah, that way round. What you see are J2000 coordinates in the ICRF, and a RA/Dec coordinate pair which is based on the orientation of the observer planet axes in space. (Switch on the grids in the view setttings dialog)  In this respect, the longitudes you should use are "ecliptical longitude J2000".

**Comment by aeromoe** [comment]
Thanks Georg...I'm going to use those coordinates.  I appreciate your reply.
Moe


-----Original Message-----
